src/online-members/script.js

```diff
@@ -10,21 +10,22 @@ function createTextNode(text) {
 
 function createProfileImage(url, alt) {
   const img = document.createElement('img');
-  img.classList = PROFILE_IMAGE_CLASS;
+  img.classList.add(...PROFILE_IMAGE_CLASS);
   img.src = member.picture
     ? getCloudinaryImgURL(url, RDS_PROFILE_IMAGE_SIZE)
     : RDS_PROFILE_DEFAULT_IMAGE;
   img.setAttribute('alt', alt);
 }
 
 function createMemberNode(member) {
-  const div = document.createElement('div');
-  div.classList = MEMBERS_CLASS_LIST;
-  div.dataset.username = member.username;
-  // div.appendChild(img);
-  div.appendChild(createTextNode(member.username));
-
-  return div;
+  const memberDiv = document.createElement('div');
```

Comment:
Make a reusable method which will take care of creating elements. This will help in reusing the method.

Reply:
Have done this [in this PR](https://github.com/Real-Dev-Squad/website-dashboard/pull/208/files#diff-0ad130b640282bfdf6bfd08eac4aa483083c7670083b149de30e0bc9fe3ba2ccR7-R12)